assets/src/components/d3/createHistogram.js

```diff
@@ -44,9 +49,21 @@ function createHistogram ({
     .domain([0, d3.max(bins, d => d.length)]).nice()
     .range([aHeight - margin.bottom, margin.top])
 
-  const svg = d3.select(domElement).append('svg')
+  // SVG Components
+  const main = d3.select(domElement).append('div')
+  // eslint-disable-next-line no-unused-vars
+  const containerGradesDiv = main.append('div')
+    .attr('aria-live', 'polite')
+    .attr('id', 'grade-view-narrative')
+    .attr('class', 'screenreader-only sr-only')
+    .text(d => {
+      return narrativeTextGrades.courseStats.concat(narrativeTextGrades.binnedGradeText, narrativeTextGrades.courseGrades.toString())
```

Comment:
Yeah, I think it probably makes sense to move this concatenation into the `grades` function.